packages/utilities/src/diff/doc/IDiffOptions.ts

```diff
@@ -21,7 +21,7 @@ export interface IDiffOptions {
      * @type {outputFormat}
      * @memberOf IDiffOptions
      */
-    outputFormat: outputFormat,
+    outputFormat?: outputFormat,
```

Comment:
What is the expected behavior when `outputFormat` is undefined?

I think if it has a default value we should document what that is in the typedoc, and if not we should leave this property as required.

Reply:
okay, default is txt